examples/hybrid_jobs/7_Running_notebooks_as_hybrid_jobs/Running_notebooks_as_hybrid_jobs.ipynb

```diff
@@ -1,400 +1,416 @@
 {
```

Comment:
<p>I'm not sure what we're showcasing here? It seems like the big addition here is the for loop around the AwsQuantumJob.create, and that seems like something we shouldn't need to show here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I'll remove the code from this section that runs the multiple jobs - you're right, it's just demonstrating a for loop, there's nothing else new here.</p>